### DISCUSSION TITLE
OneDrive Client started trying to upload files from a directory that does not exist

### DISCUSSION THREAD

**Discussion by pchad435** [abraunegg/onedrive#2878] [General]
I have been using the client for a few years. I recently start getting error messages when the client runs - OneDrive client failed to upload file ./photos/xxx.jpg. There is no folder call ./photos. There is a photos folder in the OneDrive directory that is a soft link to the directory that contains the actual photos. I'm running v2.5.0 on OpenSuse.

**Comment by abraunegg** [comment]
> I'm running v2.5.0 on OpenSuse.

Unfortunately, you are running version that has some significant bugs that were fixed , and the latest version is v2.5.2.

Please manually compile v2.5.2 or raise a bug report with OpenSuSE to provide the updated package and then re-test / re-evaluate.

If you still have an issue, please open a bug report following all the correct steps and providing a verbose debug log as per https://github.com/abraunegg/onedrive/wiki/Generate-debug-log-for-support

**Comment by pchad435** [comment]
Thanks for the help. I turned --verbose on and it did point out the problem which was files such as name.jpg vs name.JPG being seen by OneDrive as the same file. I have renamed the local files and all seems well. Since this problem only just popped up (I've been using the client for a couple of years), is it related to a change in the client in the latest release? 

Also, I had a problem with running "onedrive --synchronize --verbose > debug_output.log 2>&1" as I was unable to stop a previously running instance, or at least when I killed the process, it automatically restarted. Thinking that there was a config setting that I couldn't find, I reinstalled Tumbleweed and the behavior persists, i.e. I have not issued "systemctl enable user@onedrive.service" but as soon as I log on, onedrive starts up apparently in monitor mode, and if I kill the process, it restarts.

Thanks for your work on this, it is a great tool.

**Reply by abraunegg** [reply]
If you are using the GUI it will also be restarting the client ...

**Reply by pchad435** [reply]
I installed the GUI as an AppImage a while back but never really used it. In any event, I deleted the image and restarted. Still same behavior - onedrive is running. When I look at Task Manager, it shows a running task "onedrive --monitor". I can kill it from Task Manager, and it restarts in about 20 seconds. I'm OK with this, and just curious enough to look a bit - after some digging, found "/usr/lib/systemd/system/onedrive@.service" which has "Restart=on-failure", which explains the behavior. I assume this is something the OpenSUSE team added ?

**Reply by abraunegg** [reply]
Correct - so the package you are using is enabling / activating a default service .. so that when the client is authenticated , it just starts working.

That to me is a bug in the OpenSuse package - it should not be doing that - you should be raising a bug with OpenSuse.

**Reply by pchad435** [reply]
Raised the bug.
